assets/training/finetune_acft_hf_nlp/src/model_selector/model_selector.py

```diff
@@ -503,7 +274,49 @@ def main():
     # Adding flavor map to args
     setattr(args, "flavor_map", FLAVOR_MAP)
 
-    model_selector(args)
+    # run model selector
+    model_selector_args = model_selector(args)
+    model_name = model_selector_args.get("model_name", ModelSelectorConstants.MODEL_NAME_NOT_FOUND)
+    logger.info(f"Model name - {model_name}")
+    logger.info(f"Task name: {getattr(args, 'task_name', None)}")
+
+    # load ft config and update ACFT config
+    # finetune_config_dict = load_finetune_config(args)
+    ft_config_obj = FinetuneConfig(
+        task_name=args.task_name,
+        model_name=model_name,
+        model_type=fetch_model_type(str(Path(args.output_dir, model_name))),
+        artifacts_finetune_config_path=str(
+            Path(
+                args.pytorch_model_path or args.mlflow_model_path,
+                SaveFileConstants.ACFT_CONFIG_SAVE_PATH
+            )
+        ),
+        io_finetune_config_path=args.finetune_config_path
```

Comment:
@jpmann this is will fail if user passes huggingface id right? instead of model as input